config/checkstyle-examples-checks.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
```

Comment:
Please add:
1. File length check (15 lines? 20 lines?)
2. Some check to make sure that top level class name matches `ExampleN`

Reply:
Done. I put it at 35. If we want less, I have to shorten constantname examples.

Reply:
With the suggestion at https://github.com/checkstyle/checkstyle/pull/13218#issuecomment-1590880632 I would make it even longer for now..

Reply:
Good point, please choose number you are comfortable with.

Reply:
Update: We stay with inline, no XML will be added.

I'd stay with 35 personally. 10 lines are lost only for inline comment, package, class declaration, empty line at the end. And this is without defining properties. Sometimes I have to create a method as well just so the code can be compileable. If we add violation comments that need their own line..

Maybe 40-45, do you agree?

Reply:
Let’s do 45.

Reply:
Done.

Reply:
Also, if we are forbidding `(default)` properties in examples, we should add a module for that, per @romani ’s comment in discord.

Reply:
The problem I have with MatchXPath is that it is `TreeWalker`, wouldn't apply to non-java files.

Reply:
Good point :)